voting/orbs/Elections/harness_test.go

```diff
@@ -0,0 +1,310 @@
+// Copyright 2019 the orbs-ethereum-contracts authors
```

Comment:
1) currently in tests we turn the elections to one file - the code that does this will add any file that doesn't have _test in it
2) if/when we use the multiple file we either write the exact list of files (so each time i add one i will have to remember to add it) or do a filter (like with _test)
3) i think in the case of unit tests since they sit in the same dir its a good practice to mark all test files .
4) i wanted to the harness not to be in processs_vote_test as it serves more than one file.